src/Spaces/quadrature.jl

```diff
@@ -42,41 +42,135 @@ function quadrature_points end
 
 
 """
-    GLL{Nq}()
+    LegendreGaussLobatto
+    LGL
 
-Gauss-Legendre-Lobatto quadrature using `Nq` quadrature points.
+A Legendre-Gauss-Lobatto (LGL) quadrature rule.
+
+# Constructors
+
+    LegendreGaussLobatto(npoints=N)
+    LGL(npoints=N)
+
+Construct the quadrature rule with `N` quadrature points (`N ≥ 2`).
+
+    LegendreGaussLobatto(degree=p)
+    LGL(degree=p)
+
+Construct the quadrature rule with polynomial degree `p` (i.e. `p+1` quadrature
+points, with `p ≥ 1`).
+
+# Details
+
+A Legendre-Gauss-Lobatto rule has places quadrature points at the endpoints of
+the interval, and is able to exactly integrate polynomials of degree `2N-3`
+where `N` is the number of quadrature points.
```

Comment:
I think below the parameter is named `Nq`, so I would be consistent here. 
```suggestion
A Legendre-Gauss-Lobatto rule has places quadrature points at the endpoints of
the interval, and is able to exactly integrate polynomials of degree `2Nq-3`
where `Nq` is the number of quadrature points.
```